src/components/theme/View/View.jsx

```diff
@@ -219,12 +218,12 @@ export default class View extends Component {
 
     return (
       <div id="view">
-        <Helmet
-          bodyAttributes={{
-            class: RenderedView.displayName
+        <BodyClass
+          className={
+            RenderedView.displayName
               ? `view-${RenderedView.displayName.toLowerCase()}`
-              : 'view-undefined',
-          }}
+              : 'view-undefined'
```

Comment:
I'd have this as `view-default`, or similar, to avoid confusion with the js reserved value

Reply:
In fact it's what it is... there's some components that are not getting the displayName well, we have to investigate this... :(

It felt wrong to me to append a class that doesn't really depict the real intention of having it there... so I ended up by removing it (set it to null) so nothing shows now.

Reply:
I think it's good to have a default class for unexpected cases, it seems to make styling fallback cases more explicit than an absent class... I don't like `undefined` because it kind of points to internals of javascript and may look like the system didn't handle some error, hence my suggestion for `view-default` or similar.

But this is minor, if you're not sure you agree we can revise this in a sprint after we collect more theming experience and also ask others. Thanks!!